sway-fmt-v2/src/fmt.rs

```diff
@@ -319,4 +317,24 @@ storage {
             Formatter::format(&mut formatter, Arc::from(sway_code_to_format), None).unwrap();
         assert!(correct_sway_code == formatted_sway_code)
     }
+    #[test]
+    fn test_storage_single_line() {
+        let sway_code_to_format = r#"contract;
+
+storage {
+ long_var_name: Type1,
+      var2: Type2,
+}
+"#;
+        let correct_sway_code = r#"contract;
+
+storage { long_var_name: Type1, var2: Type2, }"#;
+        let mut config = Config::default();
+        config.structures.struct_lit_single_line = true;
+        config.whitespace.max_width = 300;
+        let mut formatter = get_formatter(config, Shape::default());
+        let formatted_sway_code =
+            Formatter::format(&mut formatter, Arc::from(sway_code_to_format), None).unwrap();
+        assert!(correct_sway_code == formatted_sway_code)
```

Comment:
Ahh I've only just noticed we've been using `assert!` in these tests when checking equality.

It's totally fine to do so for catching errors, but it's better to use `assert_eq!(a, b)` when asserting equality when possible. This is because the `assert_eq!` macro will actually display the left and right values in the failed test output, giving a better indication of why a test might be failing. On the other hand, `assert!` only shows the expression itself which is less useful.

We can probably change all of these at once in a follow-up PR though.

Reply:
I've been wrapping them in `dbg!()` when testing then removing them when finished